roles/preflight/tasks/test_run_health_check.yml

```diff
@@ -54,6 +54,14 @@
     chdir: "{{ preflight_container_artifacts.path }}"
   become: true
 
+- name: Remove the tested image
+  shell:
+    cmd: >
+      podman rmi
+      --force
+      {{ current_operator_image }}
+  become: true
```

Comment:
It would be good to know why it is needed to run as root, let's create a card for that